packages/block-library/src/social-link/edit.js

```diff
@@ -99,14 +115,14 @@ const SocialLinkEdit = ( {
 		iconBackgroundColor,
 		iconBackgroundColorValue,
 	} = context;
-	const [ showURLPopover, setPopover ] = useState( false );
 	const classes = classNames( 'wp-social-link', 'wp-social-link-' + service, {
 		'wp-social-link__is-incomplete': ! url,
```

Comment:
The way this 'is-incomplete' CSS class works should be adjusted.
When the UI shows the 'link' icon button _as a placeholder_, the button is actually an actionable control. That is fine but it looks 'disabled' because, technically, it doesn't have the url prop set.
However, if we want to use this button as an actionable, enabled, focusable control then it can't look 'disabled'.

Reply:
I think we can remove this class. With update flow, the child block is either a placeholder or matches a service. What do you think?